crates/rbuilder/src/mev_boost/mod.rs

```diff
@@ -505,7 +510,7 @@ impl RelayClient {
         Ok(builder.send().await.map_err(RelayError::RequestError)?)
     }
 
-    #[allow(clippy::too_many_arguments)]
```

Comment:
why is this no longer needed? the amount of arguments didn't change

Reply:
Probably some refactoring made it nicer and we forgot to remove it.